examples/network/custom_region/IdentityRegion.py

```diff
@@ -0,0 +1,110 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2013, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+from nupic.regions.PyRegion import PyRegion
+
+
+class IdentityRegion(PyRegion):
+  """
+  IdentityRegion is designed to implement a dummy region that returns the input
+  as the output.
+  """
+
+  def __init__(self, dataWidth, **kwargs):
+    if dataWidth <=0:
+      raise TypeError("Parameter dataWidth must be > 0")
+    
+    self.dataWidth = dataWidth
+    PyRegion.__init__(self, **kwargs)
+
+
+  #############################################################################
```

Comment:
get rid fo these comment lines and use the doc strings for any useful documentation